assets/components/src/newspack-logo/index.js

```diff
@@ -13,50 +13,53 @@ class NewspackLogo extends Component {
-								id="Shape"
-							/>
-							<Path
-								d="M492.8,31 C482.692944,31.1225404 472.668958,32.8438308 463.1,36.1 L464.7,51.8 C472.887038,48.6485236 481.534665,46.8581946 490.3,46.5 C500.2,46.5 504.5,50 504.5,60.5 L504.5,62.8 L487.5,65.1 C475.6,66.8 458.5,70.5 458.5,90.6 C458.5,105.4 469.4,112.8 479.6,112.8 C489.8,112.8 498.9,107.7 505.3,102.8 L505.8,102.8 L505.8,111.4 L524,111.4 L524,61.2 C524,38.6 511.8,31 492.8,31 Z M504.5,91.1 C499.4,94.7 493.1,98.2 487.5,98.2 C481.9,98.2 478.6,94.5 478.6,88.1 C478.6,81.7 483.7,78 488.5,77.3 L504.5,74.9 L504.5,91.1 Z"
-								id="Shape"
-							/>
-							<Path
-								d="M559.3,71.4 C559.3,53.1 569.4,46.8 581.3,46.8 C589.2,46.8 597.5,49.1 603.1,51.3 L603.1,34.8 C595.991746,32.1972725 588.469234,30.9096352 580.9,31 C555,31 539,46.8 539,72.4 C539,98 550.6,113.3 579.8,113.3 C589,113.3 596.2,111.5 604.1,108.9 L604.1,92.7 C595.2,96.2 588.4,97.5 582.2,97.5 C569.4,97.5 559.3,91.6 559.3,71.4 Z"
-								id="Path"
-							/>
-							<Path
-								d="M661.3,70.3 L693,33.1 L667.7,33.1 C643.3,63 642,65 640.3,66.8 L640.3,0.8 L620.9,0.8 L620.9,111.4 L640.3,111.4 L640.3,74.2 L670.5,111.4 L696,111.4 L661.3,70.3 Z"
-								id="Path"
-							/>
-							<Polygon
-								id="Path"
-								points="63.9 74.7 22.1 0.8 20.1 0.8 0.7 0.8 0.7 111.4 20.1 111.4 20.1 37.4 62 111.4 63.9 111.4 83.4 111.4 83.4 0.8 63.9 0.8"
-							/>
-							<Path
-								d="M136.5,30.4 C111.5,30.4 98.4,49.9 98.4,71.2 C98.4,100.4 113.8,112.8 138.7,112.8 C148.508367,112.807283 158.258204,111.289143 167.6,108.3 L167.6,92.1 C159.4,95.1 152,96.9 144.1,96.9 C129.6,96.9 119.9,92.9 119.4,75.8 L169.6,75.8 C170.016567,71.94758 170.216881,68.0748397 170.2,64.2 C170.2,47.7 161.5,30.4 136.5,30.4 Z M119.5,62.9 C120.7,52 126.6,45.1 136.4,45.1 C146.2,45.1 149.9,53.3 149.9,62.9 L119.5,62.9 Z"
-								id="Shape"
-							/>
-							<Path
-								d="M318.4,54.3 C318.4,48.4 324.8,46.1 331.8,46.1 C339.186546,46.2473017 346.519413,47.3909598 353.6,49.5 L353.6,33.4 C346.027168,31.2399092 338.173018,30.2286453 330.3,30.4 C311.5,30.4 298.3,39.3 298.3,53.8 C298.3,82.1 337.2,74.8 337.2,88.5 C337.2,95.3 331.1,97.1 322.2,97.1 C315.9,97.1 305.9,94.6 298.6,92.1 L298.6,108.1 C305.883982,111.046292 313.644357,112.639033 321.5,112.8 C339.9,112.8 357.4,107.3 357.4,87.7 C357.4,60.4 318.4,67.5 318.4,54.3 Z"
-								id="Path"
-							/>
-							<Polygon
-								id="Path"
-								points="254.8 88.7 243 33.1 225.7 33.1 212.1 88.7 200.1 33.1 178.7 33.1 199.4 111.4 222.7 111.4 234 65.4 242.3 111.4 265.7 111.4 287.7 33.1 268.1 33.1"
-							/>
-						</G>
+						{ ! compact && (
```

Comment:
This is a neat idea. I'd never realized we could programatically control different parts of SVGs with React.

Reply:
Me neither - I was happy it worked!